### DISCUSSION TITLE
Collaborative Editing of lowdefy.yaml After Docker Deployment

### DISCUSSION THREAD

**Discussion by ClorisYT** [lowdefy/lowdefy#1847] [Questions and Help]
Does Lowdefy provide an integrated development environment similar to Gitpod or Replit?

After deploying the entire project using Docker, 

how can I invite other collaborators to jointly edit the lowdefy.yaml file?

I am aware that a tutorial is available for deploying a completed project to the cloud using Docker, as detailed in Lowdefy's Docker documentation. 

However, if the goal is to share a proof of concept (POC) with collaborators, they can work on the lowdefy.yaml file together, this would necessitate rebuilding the Docker image each time. In light of this, I would like to inquire if you have any commonly used cloud environments for this purpose. For example, as a Python developer using Streamlit, I set up a Jupyter Notebook on Docker where everyone can use APIs to modify files within the project.

For Lowdefy, I am considering cloud environments such as Replit (which supports Node.js) or Gitpod. I would appreciate your opinion or experience regarding these options

**Comment by Gervwyk** [comment]
Hi @ClorisYT 
For collaboration we always use VSCode LiveShanre:
https://marketplace.visualstudio.com/items?itemName=MS-vsliveshare.vsliveshare

It's great and most devs know it. Also, usually the person creating the liveshare link will run the server. The rest ito colab is handled by git. 

Since Lowdefy simply uses yaml files, and a next.js server (running node.js), any colab environment like Replit could be set up with some effort.

Hope this helps.
